tmp.jl

```diff
@@ -1,55 +1,50 @@
 using Pkg
 Pkg.activate(".")
 # Pkg.build()
-
```

Comment:
when merging to master we could just remove `tmp.jl`

Reply:
I am not sure whether to remove it or ommit it while merging.

Reply:
if we were very picky, we shoud `git filter-branch`, but for here just ommit it